cmd/tnf/claim/show/failures/failures.go

```diff
@@ -253,12 +253,12 @@ func getFailedTestCasesByTestSuite(claimResultsByTestSuite map[string][]*claim.T
 				TestCaseDescription: tc.CatalogInfo.Description,
 			}
 
-			nonCompliantObjects, err := getNonCompliantObjectsFromFailureReason(tc.FailureReason)
+			nonCompliantObjects, err := getNonCompliantObjectsFromFailureReason(tc.SkipReason)
```

Comment:
Shouldn't this code use the new field `CheckDetails` instead?

Reply:
fixed. good catch!

Reply:
fixed. Good catch!